tests/test.py

```diff
@@ -77,10 +77,16 @@ def test_proposals_with_unknown_node_key(self):
         self.assertEqual([], data['proposals'])
 
     def test_session_stats_create_without_session_record(self):
-        re = self._post('/v1/sessions/123/stats', {
-            'bytes_sent': 20,
-            'bytes_received': 40
-        })
+        auth = generate_test_authorization()
```

Comment:
To remove duplication, this can be moved into [setUp](https://docs.python.org/2/library/unittest.html#unittest.TestCase.setUp) method to be run before running test.

Reply:
generate_test_authorization(message) will be called with different values in some tests